docs/reference/content/getting-started/quick-tour-primer.md

```diff
@@ -0,0 +1,95 @@
++++
+date = "2015-03-17T15:36:56Z"
+title = "Quick Tour Primer"
+[menu.main]
+  parent = "Getting Started"
+  identifier = "Primer"
+  weight = 20
+  pre = "<i class='fa'></i>"
++++
+
+# Quick Tour Primer
+
+The following code snippets come from the `Helpers.scala` example code
+that can be found with the [examples source]({{< srcref "examples/tour/src/main/tour/Helpers.scala">}}).
+
+{{% note %}}
+See the [installation guide]({{< relref "getting-started/installation-guide.md" >}})
+for instructions on how to install the MongoDB Scala Driver.
+{{% /note %}}
+
+## The Observable API
+
+One of the core goals of the MongoDB JVM Drivers is to be dependency free and not force users to take on a specific library dependency.
+Dependency management can cause interoperability and maintainability issues, which can be costly to maintain. 
```

Comment:
I have rewritten - I wanted to preempt the questions why don't you use "akka" or  "rx" or the "iteratees" libraries but it was rather clumsy.